finagle-core/src/main/scala/com/twitter/finagle/netty3/Netty3Transporter.scala

```diff
@@ -292,7 +294,7 @@ case class Netty3Transporter[In, Out](
         Function.const(None) _
       }
 
-      val sslHandler = new SslHandler(engine.self)
+      val sslHandler = new SslHandler(engine.self, SslHandler.getDefaultBufferPool, false, DefaultTimer, handshakeTimeout.inMillis)
```

Comment:
can you please reformat per http://docs.scala-lang.org/style/indentation.html so that the line doesn't exceed ~100 chars?

also, can you use named params to give that boolean literal some context?